Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/TabbedPage.cs

```diff
@@ -150,6 +150,48 @@ public static int GetMaxItemCount(this IPlatformElementConfiguration<Android, Fo
 			return GetMaxItemCount(config.Element);
 		}
 
+		public static readonly BindableProperty ToolbarShiftModeProperty =
```

Comment:
I wonder if we can have the same property for both, the TabbedPage and the Items, instead of having 2 different properties with different naming.

Maybe add ShiftMode to Page ?